simpleprophet/simpleprophet/pipeline.py

```diff
@@ -52,6 +54,7 @@ def update_table(
     tableref = dataset.table(table_id)
     table = bq_client.get_table(tableref)
     for product in data.keys():
+        logging.info("Processing forecasts for {}".format(product))
```

Comment:
f-strings are great, if there is no reason to use python3.5 or below. 

```suggestion
        logging.info(f"Processing forecasts for {product}")
```

Reply:
I agree f-strings would be preferred here, but this library could potentially be used in other environments and we might as well retain python 3.5 compatibility, especially as that appears to still be the default for anaconda3.